Update en.mdx to unhide old-site banner

SableRaf opened the pull request:
The old-site banner was hidden to avoid having two banners on at the same time but we still want the old-site banner to be shown when the donation banner is closed so this is not the right approach.

limzykenneth commented on src/data/examples/zh-Hans/05_Image/03_Alpha_Mask.js:
```diff
@@ -1,22 +1,22 @@
- * <p><em><span class="small"> To run this example locally, you will need two
- * image files, and a running <a href="https://github.com/processing/p5.js/wiki/Local-server">
- * local server</a>.</span></em></p>
+ * @description 在图像上加载一个蒙版来改变图像中不同位置的透明度。
```
Can we translate "mask" as "遮罩" just because that's the term MDN [uses](https://developer.mozilla.org/zh-CN/docs/Web/CSS/mask-mode)?

MuyingLi replied:
Sure. Sorry, I used the word in Photoshop. I'll check vocab. more precisely next time.

limzykenneth commented on src/data/examples/zh-Hans/05_Image/03_Alpha_Mask.js:
```diff
@@ -1,22 +1,22 @@
 /*
  * @name Alpha Mask
```
Should this be translated as well? The term "alpha" can be translated as "透明度"

MuyingLi replied:
Do you mean "透明度遮罩" for the title? I saw some examples keep the English title, but I would like to change all of them into Chinese, and we can also parenthesis the term in English, e.g. `@name 透明度遮罩(Alpha Mask)`. Since sometimes people also familiar with the word "alpha".

limzykenneth replied:
I think keeping some titles in both languages make sense. I'll leave it up to you to decide which to translate etc.

limzykenneth commented on src/data/examples/zh-Hans/05_Image/03_Alpha_Mask.js:
```diff
@@ -1,22 +1,22 @@
 
 function setup() {
   createCanvas(720, 400);
+  // maske()函数将图像蒙版覆盖在图像上
```
```suggestion
  // mask() 函数将图像蒙版覆盖在图像上
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/04_Create_Image.js:
```diff
@@ -1,20 +1,25 @@
- * @description The createImage() function provides a fresh buffer of pixels to
- * play with. This example creates an image gradient.
+ * @name 创建图像
+ * @description createImage() 函数能让我们巧妙地操控一堆像素。 此范例创建了一个渐变图像。
```
"buffer" should be translated as "缓冲" or "缓冲区".

MuyingLi replied:
Do you mean: createImage() 函数提供了一个像素缓冲区
I'm fine with direct translation, although it does not make much sense to me (probably because I translated it wrong)

limzykenneth replied:
createImage() 函数提供了一个像素缓冲区 is fine or you can still go with createImage() 函数能让我们巧妙地操控一个像素缓冲区

It's just the term 缓冲区 being used to technically and precisely denote that it is a buffer in the programming sense.

MuyingLi replied:
Got it! Thank you.

limzykenneth commented on src/data/examples/zh-Hans/05_Image/04_Create_Image.js:
```diff
@@ -1,20 +1,25 @@
 
 function setup() {
   createCanvas(720, 400);
+  // 设置图像大小230*230 像素
```
The `*` symbol should be replaced with `x` for consistency sake. Also it would be great if you can add a space before the first `230`.
```suggestion
  // 设置图像大小 230x230 像素
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/04_Create_Image.js:
```diff
@@ -1,20 +1,25 @@
   for (let x = 0; x < img.width; x++) {
     for (let y = 0; y < img.height; y++) {
       let a = map(y, 0, img.height, 255, 0);
+      // 使用set() 设置该位置像素的颜色
```
```suggestion
      // 使用 set() 设置该位置像素的颜色
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/04_Create_Image.js:
```diff
@@ -1,20 +1,25 @@
       img.set(x, y, [0, 153, 204, a]);
     }
   }
+  // 使用set() 后，必须调用updatePixels() 以使改变生效
```
```suggestion
  // 使用 set() 后，必须调用updatePixels() 以使改变生效
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/01_Background_Image.js:
```diff
@@ -1,19 +1,16 @@
-  // into the createCanvas() method. In this program, the size of
-  // the image is 720x400 pixels.
+  // 背景图像的大小必须和createCanvas() 函数中的参数一样。
+  // 该图像大小为720x400 像素。
```
```suggestion
  // 该图像大小为 720x400 像素。
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/01_Background_Image.js:
```diff
@@ -1,19 +1,16 @@
-  // The background image must be the same size as the parameters
-  // into the createCanvas() method. In this program, the size of
-  // the image is 720x400 pixels.
+  // 背景图像的大小必须和createCanvas() 函数中的参数一样。
```
```suggestion
  // 背景图像的大小必须和 createCanvas() 函数中的参数一样。
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/00_Load_and_Display_Image.js:
```diff
@@ -1,22 +1,20 @@
 
  */
-let img; // Declare variable 'img'.
+let img; // 声明变量'img'
```
```suggestion
let img; // 声明变量 'img'
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/05_Pointillism.js:
```diff
@@ -1,34 +1,43 @@
+ * <p><em><span class="small">要在本地运行此范例，您需要一个图像文件，并运行在<a href="https://github.com/processing/p5.js/wiki/Local-server">
+ * 本地伺服器</a>上。</span></em></p>
  */
+// 声明变量'img', 'smallPoint', 'largePoing'
```
```suggestion
// 声明变量 'img', 'smallPoint', 'largePoing'
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/05_Pointillism.js:
```diff
@@ -1,34 +1,43 @@
 
 function setup() {
   createCanvas(720, 400);
+  // 设置最小点宽度为4，最大点宽度为40
```
```suggestion
  // 设置最小点宽度为 4，最大点宽度为 40
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/05_Pointillism.js:
```diff
@@ -1,34 +1,43 @@
 
 function draw() {
+  // map() 函数根据鼠标水平位置，
+  // 将其在[0, 画布宽度] 的数值对应到[最小点宽度, 最大点宽度] ([4,40]) 之中，
```
```suggestion
  // 将其在 [0, 画布宽度] 的数值对应到 [最小点宽度, 最大点宽度] ([4,40]) 之中，
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/05_Pointillism.js:
```diff
@@ -1,34 +1,43 @@
+  // 将其在[0, 画布宽度] 的数值对应到[最小点宽度, 最大点宽度] ([4,40]) 之中，
+  // 对应的数值即为点的大小
   let pointillize = map(mouseX, 0, width, smallPoint, largePoint);
+  // 随机生成坐标(x, y)
```
```suggestion
  // 随机生成坐标 (x, y)
```

limzykenneth commented on src/data/examples/zh-Hans/05_Image/05_Pointillism.js:
```diff
@@ -1,34 +1,43 @@
+  // 随机生成坐标(x, y)
   let x = floor(random(img.width));
   let y = floor(random(img.height));
+  // 得到图像中坐标(x, y) 的颜色
```
```suggestion
  // 得到图像中坐标 (x, y) 的颜色
```

MuyingLi commented:
@limzykenneth  Just to summary the format: **add a space before 'quote', number, (parenthesis), [bracket], english word, and function()** . Do I miss any other format? I will follow these rules next time. Thanks again for correcting each and every space!

limzykenneth commented:
Basically the "rule" is space between anything that is not a full width character and those that are full width character. However there are exceptions to these rules particularly around full width characters that visually doesn't take up that much space `。` for example, if a non full width character comes after `。` there is not need to add an additional space. 

I have a library on the website to take care of this automatically where possible but for code comments we'll need to do them manually.

MuyingLi commented:
Ok. I just corrected everything and I think they are ready to be put on the website. Thanks again for such quick response! @limzykenneth

limzykenneth commented on src/data/examples/zh-Hans/07_Color/03_Color_Variables.js:
```diff
@@ -12,8 +12,8 @@ function setup() {
 
-  // These statements are equivalent to the statements above.
-  // Programmers may use the format they prefer.
+  // 以下陈述与上面陈述的等效。
```
I find translating "statement" as "陈述" a bit weird. I have "control statement" translated as "控制语句" in the glossary but I'm not super firm on translating "statement" as "语句". It may just be me though, what do you think?

limzykenneth commented on src/data/examples/zh-Hans/07_Color/05_Linear_Gradient.js:
```diff
@@ -1,17 +1,16 @@
- * @description The lerpColor() function is useful for interpolating between
- * two colors.
+ * @name 线性渐变
+ * @description lerpColor()函数可用于在两个颜色之间插值。
```
```suggestion
 * @description lerpColor() 函数可用于在两个颜色之间插值。
```

limzykenneth commented on src/data/examples/zh-Hans/07_Color/07_Lerp_Color.js:
```diff
@@ -1,7 +1,6 @@
 /*
  * @name Lerp Color
```
I think the title can be translated here.

limzykenneth commented on src/data/examples/zh-Hans/07_Color/07_Lerp_Color.js:
```diff
@@ -18,31 +17,47 @@ function draw() {
+      random(-40, 220),
+      random(height),
+      random(-40, 220),
+      random(height)
```
Can the formatting of these stay the same? It just makes the syntax of `quad()` a bit easier to read.

limzykenneth commented:
When you are all done with this PR please let me know, I'll merge it whenever you are ready.

sm7515 commented:
The latest commit resolves all comments : ) 

If the future, we want to translate other example sections, we can just push to this translate branch right?

limzykenneth commented:
@sm7515 Depends on you really. If you want to keep working on this branch we can keep the PR open a bit longer. Or we can merge this PR then a new one can be opened from the same branch against master branch here.

sm7515 commented:
Gotcha. I think we can merge and later on for new translations, we’ll open a new pr. Thank you.

limzykenneth commented:
Thanks for all your work @MuyingLi and @sm7515!

SableRaf committed:
Update en.mdx to unhide old-site banner

The old-site banner was hidden to avoid having two banner on at the same time but we still want the old-site banner to be shown when the donation banner is closed so this is not the right approach.
